ChainSafe/web3.unity
# IPFS upload fix

IPFS upload fixed
Users can now upload an image or metadata to storage or both at the same time.
Cleaned up and abstracted classes to match our current classes.
Models for metadata match standards but will also work with the SDK, will update with marketplace PR if needed.

![image](https://github.com/ChainSafe/web3.unity/assets/57473220/68bff7d7-b541-4aa5-bf81-807949638209)


To Test:
Load up the sample scene & scroll to ipfs.
Enter your bucket & API key from storage https://app.storage.chainsafe.io/ into the IPFS sample object on the left.
The metadata is taken from the object in the sample scene and images are prompted for upload at runtime.
Test image, metadata & image & metadata functions.

Result:
Metadata json and images should be reachable via the link returned from the call in the console as well as being visible in the storage bucket dashboard.

closes #948
